lib/src/comment_references/parser.dart

```diff
@@ -76,11 +77,11 @@ final StringTrie operatorParseTrie = () {
 // TODO(jcollins-g): align with [CommentReference] from analyzer AST.
```

Comment:
Hopefully, we won't need this parser in the future. All of this work is so complicated and the analyzer already does it.

Reply:
Yes yes yes :D